frontend/app/utils/env.server.ts

```diff
@@ -87,6 +87,12 @@ const serverEnv = z.object({
 
   // CCT get PDF settings
   CCT_VAULT_COMMUNITY: z.string().default('community_default'),
+
+  //Memo cache max age
+  MEMO_MAXAGE_ALLPREFERREDLANGUAGES: z.coerce.number().optional(),
+  MEMO_MAXAGE_PREFERREDLANGUAGE: z.coerce.number().optional(),
+  MEMO_MAXAGE_ALLCOUNTRIES: z.coerce.number().optional(),
+  MEMO_MAXAGE_ALLREGIONS: z.coerce.number().optional(),
```

Comment:
I think you should provide sane defaults here. If we forget to set these config values in production, we're going to hammer the downstream service.

Reply:
removed optional and added default of 1 hr (still includes undefined as response type)